docusaurus/docs/_linux-package-repo.mdx

```diff
@@ -0,0 +1,45 @@
+
+import Details from '@theme/MDXComponents/Details';
+
+<Details>
+<summary>Manually set up DEB repository</summary>
```

Comment:
I think it's worthwhile to add Ubuntu/Fedora/RedHat onto these too. I think people will naturally search for "how to install openziti on fedora" not "by rpm". At least the most popular ones like Ubuntu/Fedora/RedHat?